core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -67,24 +78,35 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
     producer.close(60, TimeUnit.SECONDS)
   }
 
+  def produceMessages(topic: String, range: Range) = {
+    Source(range)
```

Comment:
`Source(range)` is not async? it just returns a `Source` which you later have to connect to a `Sink` and then `run`, so I see no need for `Future` here